scripts/flatpak/chiaki4deck.yaml

```diff
@@ -204,6 +204,20 @@ modules:
       url: https://gitlab.xiph.org/xiph/speexdsp.git
       tag: SpeexDSP-1.2.1
 
+  - name: wireless-tools
```

Comment:
You can just use `dbus-send` to communicate with NetworkManager instead of pulling yet another dependency.